README.md

```diff
@@ -94,14 +94,14 @@ module.exports = lambdaCfn.build({
 });
 ```
 
-If you'd like to create a Lambda function that runs on the older Node.js 4.3 runtime with a memory size of 256 MB and a timeout of 120 seconds:
+If you'd like to create a Lambda function that runs on the newer Node.js 10.x runtime with a memory size of 256 MB and a timeout of 120 seconds:
```

Comment:
Since nodejs 10.x is now the default, this should use `nodejs8.10` and read similar to the old language.